data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml

```diff
@@ -14,7 +14,7 @@ terraform:
     # GENERAL VARIABLES #
     az_region: '%PUBLIC_CLOUD_REGION%'
     admin_user: 'cloudadmin'
-    deployment_name: '%PUBLIC_CLOUD_RESOURCE_GROUP%'
+    deployment_name: '%QESAP_DEPLOYMENT_NAME%'
```

Comment:
Seems like I forget this one in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17079

Thanks